integration.go

```diff
@@ -43,7 +50,20 @@ type AWSIntegrationInput struct {
 	OwnershipTagKeys     []string `json:"ownershipTagKeys"`
 }
 
+type IntegrationInput struct {
+	Id   								*string     `json:"id,omitempty"`
+	Alias   						*string     `json:"alias,omitempty"`
+}
+
+type NewRelicIntegrationInput struct {
+	Integration 			IntegrationInput  `graphql:"integration"`
```

Comment:
I feel I am doing this incorrectly because in my tests I get errors that its looking for `integration` and not `Integration`

<img width="1411" alt="image" src="https://github.com/OpsLevel/opslevel-go/assets/5561612/9011f6c0-a789-47aa-9c3d-19c8a4d0ddd4">

Reply:
https://github.com/OpsLevel/opslevel-go/actions/runs/5256432767/jobs/9497750616?pr=207#step:5:260

Reply:
ahh - thats because you are nesting it under a field on `NewRelicIntegrationInput` - if you want to achieve something like inheritance (but golong doesn't have inheritance) you need to use composition

```
type NewRelicIntegrationInput struct {
    IntegrationInput
	ApiKey           	*string  					`json:"apiKey,omitempty"`
	BaseUrl 					*string  					`json:"baseUrl,omitempty"`
	AccountKey     		*string  					`json:"accountKey,omitempty"`
}
```


Is what you'd do - but its going to make it harder to work with.  I'd just put the fields for ID and Alias directly into `NewRelicIntegrationInput` 

Go is a very "explicit" language compared to Ruby.